### ISSUE TITLE
FR: Add configurable atomic variables

### ISSUE THREAD

**Issue by bengolds** [arnog/mathlive#1037]
tl;dr: Is it possible to set certain strings as "Atomic" so that they can be deleted quickly?

## Description

We're working with custom keyboards so that users can solve algebra problems on their mobile devices (or on Desktop, if they don't know how to type certain characters). One of the great advantages with the custom keyboards is making it easy for users to type in variables with subscripts, like `v_0` or `A_x`.

<img width="642" alt="CleanShot 2021-06-21 at 14 57 48@2x" src="https://user-images.githubusercontent.com/446007/122833101-396dd900-d2a1-11eb-928f-8c62fabe31a3.png">

However, deleting those characters is a bit annoying and error prone, as user have to press delete ~3 times. It'd be nicer if we could specify a list of Latex strings/variables that can be treated as atomic and can be deleted with one key.

**Comment by arnog** [comment]
There are two ways to achieve this:

1/ In the virtual keyboard definition, instead of `A_x` for example, use `\mathop{A_x}`. The `\mathop{}` command is treated as atomic and its content won't be selectable.

2/ Define a macro (https://cortexjs.io/mathlive/guides/macros/) and use it in the virtual keyboard: macros are also treated as atomic elements.

**Comment by bengolds** [comment]
Fantastic! Thanks Arno; I'll give these a shot and let you know if they work. Does `\mathop` imply any other changes to the behavior/appearance of the atoms?

**Comment by arnog** [comment]
@bengolds actually, it does. 

It implies thats this is an `operator` (like `+`), in the TeX sense, which will affect the layout (specifically the amount of space left and right of it). 

I think for your use case `\mathord{}` would be more appropriate: an `ord` in the TeX sense is an "ordinary" atom, which is used for variables, so that would be a good fit.
